recipes/phasecalculator/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "phasecalculator" %}
+{% set version = "1.0.4.b" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
I've made some changes and done a 1.0.5, so it is now a full release.